pom.xml

```diff
@@ -60,6 +60,7 @@
         <module>features/org.wso2.carbon.identity.oauth.server.feature</module>
         <module>features/org.wso2.carbon.identity.oauth.ui.feature</module>
         <module>features/org.wso2.carbon.identity.oauth.dcr.server.feature</module>
+        <module>components/org.wso2.carbon.identity.oauth.ciba</module>
```

Comment:
Lets move this up, lets add after
        <module>components/org.wso2.carbon.identity.oauth.client.authn.filter</module>

Reply:
Refactored.